test/house_test.rb

```diff
@@ -0,0 +1,66 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/room"
+require "./lib/house"
+require "pry"
+
+class HouseTest < Minitest::Test
+
+  def setup
+    @house = House.new("$400000", "123 sugar lane")
+    @room_1 = Room.new(:bedroom, 10, '13')
+    @room_2 = Room.new(:bedroom, 11, '15')
+    @room_3 = Room.new(:living_room, 25, '15')
+    @room_4 = Room.new(:basement, 30, '41')
+  end
+
+  def test_it_exists
+    assert_instance_of House, @house
+  end
+
+  def test_it_has_a_price
+    assert_equal 400000, @house.price
+  end
+
+  def test_it_has_an_address
+    assert_equal "123 sugar lane", @house.address
+  end
+
+  def test_it_start_with_no_rooms
+    assert_equal [], @house.rooms
+  end
+
+  def test_it_can_add_rooms
+    @house.add_room(@room_1)
+    @house.add_room(@room_2)
+    assert_equal [@room_1, @room_2], @house.rooms
+  end
+
+  def test_it_can_be_below_market_average
+    assert_equal false, @house.above_market_average?
```

Comment:
For any method that could return true or false, I would write a test for both (this might have shown your error in the method)